css/media.css

```diff
@@ -395,3 +396,216 @@
     margin-left: 4px;
   }
 }
+
+@media (max-width: 1360px) {
+  .aside__footer {
+  padding-right: 26px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5948685)